command/meta.go

```diff
@@ -84,6 +89,12 @@ func (m *Meta) Client() (*api.Client, error) {
 	if m.flagAddress != "" {
 		config.Address = m.flagAddress
 	}
+	if v := os.Getenv(EnvNomadRegion); v != "" {
```

Comment:
It is actually distinct from flagAddress. The precedent is the env-var -> flag -> job's region in the case of run and plan